python/labbox_ephys/import_to_workspace.py

```diff
@@ -75,32 +75,39 @@ def _import_le_sorting(sortings_subfeed: kp.Subfeed, le_sorting):
         }
     })
 
-def import_recording(*, feed: kp.Feed, workspace_name: str, recording: se.RecordingExtractor, recording_id: str):
+def random_id():
+    return str(uuid.uuid4())[-12:]
+
+def import_recording(*, feed: kp.Feed, workspace_name: str, recording: se.RecordingExtractor, recording_label: str):
+    recording_id = 'R-' + random_id()
```

Comment:
Now, when importing a recording, the recording ID is randomly generated. The recording label is what displays on the GUI.